docs/instructions.md

```diff
@@ -75,10 +74,44 @@ After following the steps in [Quick Start](#quick-start-app) above:
 
 ### App Settings ###
 
+#### iOS
+
+To change your general settings for the Whisper app, click the Settings button on the start screen. You will be automatically redirected to the Whisper section of your device's Settings app, which contains two lists of options. The first, "Allow Whisper to Access," lets you control how the app interacts with other features on your device. The second, "Whisper Settings," lets you control how the app itself behaves. 
+
+**Allow Whisper to Access** has four sections:
+
+* **Bluetooth** — allowing Whisper access to Bluetooth is what lets you start and join in-person conversations automatically when you tap Whisper or Listen. We highly recommend allowing Bluetooth access!
+* **Siri & Search** — tapping this option brings you to a menu where you can refine how much access Apple has to how you use Whisper, as well as whether you can use the built-in voice command functions to find and open Whisper
+* **Notifications** — tapping this option brings you to a menu where you can change what notifications (if any) the Whisper app will give you, the style of those notifications, and whether you'll see whispers on your lock screen
+  * This is where you choose whether the app alerts you with a sound when a new whisper comes in; to choose a particular alert sound, go to "Alert Sound" in Whisper Settings
```

Comment:
Actually, it's the Whisperer who chooses the alert sound.  The "Alert Sound" setting is only used when Whispering (to set the initial sound).  Now that Whisperers can select this sound in the Whisper screen, we might want to remove the "Alert Sound" setting completely because it's confusing for Listeners to have one that does nothing :).  Let me know what you think.

Reply:
I think it makes the most sense to remove this pref completely, and have Whisperers set their alert sound from the Whisperer settings (see issue #26)

Reply:
Suggested rewrite: "This is where you choose whether the app alerts you with a sound when a new whisper comes in while the app is running in the background. We recommend keeping this on. Note that the Whisperer is the one who chooses what alert sound their whispers make.